src/universal/common/components/AppHeader/AppHeader.tsx

```diff
@@ -9,7 +9,7 @@ export default class AppHeader extends React.Component<any, any> {
     return (
       <div className={css.AppHeader}>
         <div className={css.LogoContainer}>
-          <a href="/"><AppLogo /></a>
+          <a href="https://www.everreal.de"><AppLogo /></a>
```

Comment:
Yes, damn, good catch, it should be `.co`. Btw `everreal.de` is pointed to redirect to `everreal.co`, but `https://www.everreal.de` fails to load as far as I see now.